locales/en-US/app.ftl

```diff
@@ -1,59 +1,72 @@
 siteName = Firefox Test Pilot
 
+# Page titles, put in the <title> HTML tag.
+[pageTitle]
```

Comment:
Updated, and the new version is [parsing correctly](http://l20n.github.io/tinker/#eyJzb3VyY2UiOiJzaXRlTmFtZSA9IEZpcmVmb3ggVGVzdCBQaWxvdFxuXG4jIFBhZ2UgdGl0bGVzLCBwdXQgaW4gdGhlIDx0aXRsZT4gSFRNTCB0YWcuXG5bW3BhZ2VUaXRsZV1dXG5wYWdlVGl0bGVEZWZhdWx0ID0gRmlyZWZveCBUZXN0IFBpbG90XG5wYWdlVGl0bGVMYW5kaW5nUGFnZSA9IEZpcmVmb3ggVGVzdCBQaWxvdFxucGFnZVRpdGxlRXhwZXJpbWVudExpc3RQYWdlID0gRmlyZWZveCBUZXN0IFBpbG90IC0gRXhwZXJpbWVudHNcbnBhZ2VUaXRsZUV4cGVyaW1lbnQgPSBGaXJlZm94IFRlc3QgUGlsb3QgLSB7JHRpdGxlfVxuXG4jIExpbmtzIGluIHRoZSBmb290ZXIuXG5bW2Zvb3RlckxpbmtdXVxuZm9vdGVyTGlua0Nvb2tpZXMgPSBDb29raWVzXG5mb290ZXJMaW5rUHJpdmFjeSA9IFByaXZhY3lcbmZvb3RlckxpbmtUZXJtcyA9IFRlcm1zXG5mb290ZXJMaW5rTGVnYWwgPSBMZWdhbFxuZm9vdGVyTGlua0Fib3V0ID0gQWJvdXQgVGVzdCBQaWxvdFxuXG4jIEl0ZW1zIGluIHRoZSBtZW51LlxuW1ttZW51XV1cbmhvbWUgPSBIb21lXG5tZW51VGl0bGUgPSBTZXR0aW5nc1xubWVudVdpa2kgPSBUZXN0IFBpbG90IFdpa2lcbm1lbnVEaXNjdXNzID0gRGlzY3VzcyBUZXN0IFBpbG90XG5tZW51RmlsZUlzc3VlID0gRmlsZSBhbiBJc3N1ZVxubWVudVJldGlyZSA9IFVuaW5zdGFsbCBUZXN0IFBpbG90XG5cbiMgVGhlIHNwbGFzaCBvbiB0aGUgaG9tZXBhZ2UuXG5bW2xhbmRpbmddXVxubGFuZGluZ0ludHJvTGVhZCA9IEdvIGJleW9uZCAuIC4gLlxubGFuZGluZ0ludHJvT25lID0gVGVzdCBuZXcgZmVhdHVyZXMuXG5sYW5kaW5nSW50cm9Ud28gPSBHaXZlIHlvdXIgZmVlZGJhY2suXG5sYW5kaW5nSW50cm9UaHJlZSA9IEhlbHAgYnVpbGQgRmlyZWZveC5cbmxhbmRpbmdMZWdhbE5vdGljZSA9IEJ5IHByb2NlZWRpbmcsIHlvdSBhZ3JlZSB0byB0aGUgPGE+VGVybXMgb2YgVXNlPC9hPiBhbmQgPGE+UHJpdmFjeSBOb3RpY2U8L2E+IG9mIFRlc3QgUGlsb3QuXG5sYW5kaW5nRXhwZXJpbWVudHNUaXRsZSA9IFRyeSBvdXQgdGhlIGxhdGVzdCBleHBlcmltZW50YWwgZmVhdHVyZXNcblxuIyBSZWxhdGVkIHRvIHRoZSBpbnN0YWxsYXRpb24gb2YgdGhlIFRlc3QgUGlsb3QgYWRkLW9uLlxuW1tsYW5kaW5nSW5zdGFsbF1dXG5sYW5kaW5nSW5zdGFsbEJ1dHRvbiA9IEluc3RhbGwgdGhlIFRlc3QgUGlsb3QgQWRkLW9uXG5sYW5kaW5nSW5zdGFsbGluZ0J1dHRvbiA9IEluc3RhbGxpbmcuLi5cbmxhbmRpbmdJbnN0YWxsZWRCdXR0b24gPSBDaG9vc2UgeW91ciBmZWF0dXJlc1xuXG4jIEhvbWVwYWdlIG1lc3NhZ2luZyBmb3IgdXNlcnMgbm90IG9uIEZpcmVmb3ggb3Igd2l0aCBhbiBvbGQgdmVyc2lvbiBvZiBGaXJlZm94LlxuW1tsYW5kaW5nRmlyZWZveF1dXG5sYW5kaW5nUmVxdWlyZXNEZXNrdG9wID0gVGVzdCBQaWxvdCByZXF1aXJlcyBGaXJlZm94IGZvciBEZXNrdG9wIG9uIFdpbmRvd3MsIE1hYyBvciBMaW51eFxubGFuZGluZ0Rvd25sb2FkRmlyZWZveERlc2MgPSAoVGVzdCBQaWxvdCBpcyBhdmFpbGFibGUgZm9yIEZpcmVmb3ggb24gV2luZG93cywgT1MgWCBhbmQgTGludXgpXG5sYW5kaW5nVXBncmFkZURlc2MgPSBUZXN0IFBpbG90IHJlcXVpcmVzIEZpcmVmb3ggNDUgb3IgaGlnaGVyLlxubGFuZGluZ0Rvd25sb2FkRmlyZWZveFRpdGxlID0gRmlyZWZveFxubGFuZGluZ1VwZ3JhZGVGaXJlZm94VGl0bGUgPSBVcGdyYWRlIEZpcmVmb3hcbmxhbmRpbmdEb3dubG9hZEZpcmVmb3hTdWJUaXRsZSA9IEZyZWUgRG93bmxvYWRcblxuIyBBIHNlY3Rpb24gb2YgdGhlIGhvbWVwYWdlIGV4cGxhaW5pbmcgaG93IFRlc3QgUGlsb3Qgd29ya3MuXG5bW2xhbmRpbmdDYXJkXV1cbmxhbmRpbmdDYXJkTGlzdFRpdGxlID0gR2V0IHN0YXJ0ZWQgaW4gMywgMiwgMVxubGFuZGluZ0NhcmRPbmUgPSBHZXQgdGhlIFRlc3QgUGlsb3QgYWRkLW9uXG5sYW5kaW5nQ2FyZFR3byA9IEVuYWJsZSBleHBlcmltZW50YWwgZmVhdHVyZXNcbmxhbmRpbmdDYXJkVGhyZWUgPSBUZWxsIHVzIHdoYXQgeW91IHRoaW5rXG5cbiMgU2hvd24gYWZ0ZXIgdGhlIHVzZXIgaW5zdGFsbHMgdGhlIFRlc3QgUGlsb3QgYWRkLW9uLlxuW1tvbmJvYXJkaW5nXV1cbm9uYm9hcmRpbmdNZXNzYWdlID0gV2UgcHV0IGFuIGljb24gaW4geW91ciB0b29sYmFyIHNvIHlvdSBjYW4gYWx3YXlzIGZpbmQgVGVzdCBQaWxvdC5cblxuIyBFcnJvciBtZXNzYWdlIHBhZ2VzLlxuW1tlcnJvcl1dXG5lcnJvckhlYWRpbmcgPSBXaG9vcHMhXG5lcnJvck1lc3NhZ2UgPSBMb29rcyBsaWtlIHdlIGJyb2tlIHNvbWV0aGluZy4gPGJyPiBNYXliZSB0cnkgYWdhaW4gbGF0ZXIuXG5ub3RGb3VuZEhlYWRlciA9IEZvdXIgT2ggRm91ciFcblxuIyBBIG1vZGFsIHByb21wdCB0byBzaWduIHVwIGZvciB0aGUgVGVzdCBQaWxvdCBuZXdzbGV0dGVyLlxuW1tlbWFpbE9wdEluXV1cbmVtYWlsT3B0SW5EaWFsb2dUaXRsZSA9IFdlbGNvbWUgdG8gVGVzdCBQaWxvdCFcbmVtYWlsT3B0SW5NZXNzYWdlID0gRmluZCBvdXQgYWJvdXQgbmV3IGV4cGVyaW1lbnRzIGFuZCBzZWUgdGVzdCByZXN1bHRzIGZvciBleHBlcmltZW50cyB5b3UndmUgdHJpZWQuXG5lbWFpbFZhbGlkYXRpb25FcnJvciA9IFBsZWFzZSB1c2UgYSB2YWxpZCBlbWFpbCBhZGRyZXNzIVxuIyBMT0NBTElaQVRJT04gTk9URTogVGhlICc6KScgY2hhcmFjdGVycyBpbiB0aGUgZW1haWxPcHRJbklucHV0IHBsYWNlaG9sZGVyIGFyZSBhIHNtaWxleSBmYWNlIGVtb3RpY29uLlxuZW1haWxPcHRJbklucHV0ID1cbiAgW2h0bWwvcGxhY2Vob2xkZXJdIGVtYWlsIGdvZXMgaGVyZSA6KVxuZW1haWxPcHRJbkJ1dHRvbiA9IFNpZ24gbWUgdXBcbmVtYWlsT3B0SW5Ta2lwID0gU2tpcFxuZW1haWxPcHRJbkNvbmZpcm1hdGlvblRpdGxlID0gRW1haWwgU2VudFxuZW1haWxPcHRJblN1Y2Nlc3NNZXNzYWdlMiA9IFRoYW5rIHlvdSFcbmVtYWlsT3B0SW5Db25maXJtYXRpb25DbG9zZSA9IE9uIHRvIHRoZSBleHBlcmltZW50cy4uLlxuXG4jIEEgbGlzdGluZyBvZiBhbGwgVGVzdCBQaWxvdCBleHBlcmltZW50cy5cbltbZXhwZXJpbWVudHNMaXN0XV1cbmV4cGVyaW1lbnRMaXN0UGFnZUhlYWRlciA9IFJlYWR5IGZvciBUYWtlb2ZmIVxuZXhwZXJpbWVudExpc3RQYWdlU3ViSGVhZGVyICA9IFBpY2sgdGhlIGZlYXR1cmVzIHlvdSB3YW50IHRvIHRyeS4gPGJyPiBDaGVjayBiYWNrIHNvb24gZm9yIG1vcmUgZXhwZXJpbWVudHMuXG5leHBlcmltZW50TGlzdEVuYWJsZWRUYWIgPSBFbmFibGVkXG5leHBlcmltZW50TGlzdEp1c3RMYXVuY2hlZFRhYiA9IEp1c3QgTGF1bmNoZWRcbmV4cGVyaW1lbnRMaXN0SnVzdFVwZGF0ZWRUYWIgPSBKdXN0IFVwZGF0ZWRcbmV4cGVyaW1lbnRMaXN0RW5kaW5nVG9tb3Jyb3cgPSBFbmRpbmcgVG9tb3Jyb3dcbmV4cGVyaW1lbnRMaXN0RW5kaW5nU29vbiA9IEVuZGluZyBTb29uXG5cbiMgQW4gaW5kaXZpZHVhbCBleHBlcmltZW50IGluIHRoZSBsaXN0aW5nIG9mIGFsbCBUZXN0IFBpbG90IGV4cGVyaW1lbnRzLlxuW1tleHBlcmltZW50Q2FyZF1dXG5leHBlcmltZW50Q2FyZE1hbmFnZSA9IE1hbmFnZVxuZXhwZXJpbWVudENhcmRHZXRTdGFydGVkID0gR2V0IFN0YXJ0ZWRcbmV4cGVyaW1lbnRDYXJkTGVhcm5Nb3JlID0gTGVhcm4gTW9yZVxuXG4jIEEgbW9kYWwgcHJvbXB0IHNob3duIHdoZW4gYSB1c2VyIGRpc2FibGVzIGFuIGV4cGVyaW1lbnQuXG5bW2ZlZWRiYWNrXV1cbmZlZWRiYWNrU3VibWl0QnV0dG9uID0gVGFrZSBhIHF1aWNrIHN1cnZleVxuZmVlZGJhY2tDYW5jZWxCdXR0b24gPSBDbG9zZVxuZmVlZGJhY2tVbmluc3RhbGxUaXRsZSA9IFRoYW5rIFlvdSFcbmZlZWRiYWNrVW5pbnN0YWxsQ29weSA9XG4gICAgfCBZb3VyIHBhcnRpY2lwYXRpb24gaW4gRmlyZWZveCBUZXN0IFBpbG90IG1lYW5zXG4gICAgfCBhIGxvdCEgUGxlYXNlIGNoZWNrIG91dCBvdXIgb3RoZXIgZXhwZXJpbWVudHMsXG4gICAgfCBhbmQgc3RheSB0dW5lZCBmb3IgbW9yZSB0byBjb21lIVxuXG4jIEEgbW9kYWwgcHJvbXB0IHRlbGxpbmcgYSB1c2VyIHRoYXQgdGhleSBhcmUgYWJvdXQgdG8gZ28gdG8gYW4gZXh0ZXJuYWwgZm9ydW0gZm9yIGRpc2N1c3Npb24uXG5bW2Rpc2N1c3NOb3RpZnldXVxuZGlzY3Vzc05vdGlmeVRpdGxlID0gSnVzdCBvbmUgc2Vjb25kLi4uXG5kaXNjdXNzTm90aWZ5TWVzc2FnZUFjY291bnRsZXNzID1cbiAgICB8IDxwPkluIHRoZSBzcGlyaXQgb2YgZXhwZXJpbWVudGF0aW9uLCB3ZSBhcmUgdXNpbmcgYW4gZXh0ZXJuYWwgZm9ydW0gc2VydmljZS5cbiAgICB8IFlvdSB3aWxsIG5lZWQgdG8gY3JlYXRlIGFuIGFjY291bnQgaWYgeW91XG4gICAgfCB3aXNoIHRvIHBhcnRpY2lwYXRlIG9uIHRoZSBmb3J1bXMuPC9wPlxuICAgIHwgPHA+SWYgeW91IGRvbid0IHdhbnQgdG8gY3JlYXRlIGFuIGFjY291bnQsIHlvdSBjYW5cbiAgICB8IGFsd2F5cyBsZWF2ZSBmZWVkYmFjayB0aHJvdWdoIFRlc3QgUGlsb3QuXG4gICAgfCA8YnI+XG4gICAgfCAoV2UgcmVhbGx5IGRvIHJlYWQgdGhpcyBzdHVmZik8L3A+XG5kaXNjdXNzTm90aWZ5U3VibWl0QnV0dG9uID0gVGFrZSBtZSB0byB0aGUgZm9ydW1cbmRpc2N1c3NOb3RpZnlDYW5jZWxCdXR0b24gPSBDYW5jZWxcblxuIyBBIG1vZGFsIHByb21wdCBzaG93biBiZWZvcmUgdGhlIGZlZWRiYWNrIHN1cnZleSBmb3Igc29tZSBleHBlcmltZW50cy5cbltbZXhwZXJpbWVudFByZUZlZWRiYWNrXV1cbmV4cGVyaW1lbnRQcmVGZWVkYmFja1RpdGxlID0geyR0aXRsZX0gZmVlZGJhY2tcbmV4cGVyaW1lbnRQcmVGZWVkYmFja0xpbmtDb3B5ID0gR2l2ZSBmZWVkYmFjayBhYm91dCB0aGUgeyR0aXRsZX0gZXhwZXJpbWVudFxuXG4jIEEgc3BsYXNoIHNob3duIG9uIHRvcCBvZiB0aGUgZXhwZXJpbWVudCBwYWdlIHdoZW4gVGVzdCBQaWxvdCBpcyBub3QgaW5zdGFsbGVkLlxuW1tleHBlcmltZW50UHJvbW9dXVxuZXhwZXJpbWVudFByb21vSGVhZGVyID0gUmVhZHkgZm9yIFRha2VvZmY/XG5leHBlcmltZW50UHJvbW9TdWJoZWFkZXIgPSBXZSdyZSBidWlsZGluZyBuZXh0LWdlbmVyYXRpb24gZmVhdHVyZXMgZm9yIEZpcmVmb3guIEluc3RhbGwgVGVzdCBQaWxvdCB0byB0cnkgdGhlbSFcblxuIyBUaGUgZXhwZXJpbWVudCBkZXRhaWwgcGFnZS4gXG5bW2V4cGVyaW1lbnRQYWdlXV1cbmlzRW5hYmxlZFN0YXR1c01lc3NhZ2UgPSB7JHRpdGxlfSBpcyBlbmFibGVkLlxuaW5zdGFsbEVycm9yTWVzc2FnZSA9IFVoIG9oLiB7JHRpdGxlfSBjb3VsZCBub3QgYmUgZW5hYmxlZC4gVHJ5IGFnYWluIGxhdGVyLlxucGFydGljaXBhbnRDb3VudCA9IDxzcGFuPnskaW5zdGFsbGF0aW9uX2NvdW50fTwvc3Bhbj4gcGFydGljaXBhbnRzXG5vdGhlckV4cGVyaW1lbnRzID0gVHJ5IG91dCB0aGVzZSBleHBlcmltZW50cyBhcyB3ZWxsXG5naXZlRmVlZGJhY2sgPSBHaXZlIEZlZWRiYWNrXG5kaXNhYmxlSGVhZGVyID0gRGlzYWJsZSBFeHBlcmltZW50P1xuZGlzYWJsZUV4cGVyaW1lbnQgPSBEaXNhYmxlIHskdGl0bGV9XG5kaXNhYmxlRXhwZXJpbWVudFRyYW5zaXRpb24gPSBEaXNhYmxpbmcuLi5cbmVuYWJsZUV4cGVyaW1lbnQgPSBFbmFibGUgeyR0aXRsZX1cbmVuYWJsZUV4cGVyaW1lbnRUcmFuc2l0aW9uID0gRW5hYmxpbmcuLi5cbm1lYXN1cmVtZW50cyA9IFlvdXIgcHJpdmFjeVxuZXhwZXJpbWVudFByaXZhY3lOb3RpY2UgPSBZb3UgY2FuIGxlYXJuIG1vcmUgYWJvdXQgdGhlIGRhdGEgY29sbGVjdGlvbiBmb3IgeyR0aXRsZX0gaGVyZS5cbmNvbnRyaWJ1dG9yc0hlYWRpbmcgPSBCcm91Z2h0IHRvIHlvdSBieVxudmVyc2lvbiA9IFZlcnNpb25cbmNoYW5nZWxvZyA9IGNoYW5nZWxvZ1xudG91ckxpbmsgPSB0b3VyXG5sYXN0VXBkYXRlID0gTGFzdCBVcGRhdGVcbmNvbnRyaWJ1dGUgPSBDb250cmlidXRlXG5idWdSZXBvcnRzID0gQnVnIFJlcG9ydHNcbmRpc2NvdXJzZSA9IERpc2NvdXJzZVxudG91ck9uYm9hcmRpbmdUaXRsZSA9IHskdGl0bGV9IGVuYWJsZWQhXG50b3VyRG9uZUJ1dHRvbiA9IERvbmVcbnVzZXJDb3VudENvbnRhaW5lciA9IFRoZXJlIGFyZSA8c3Bhbj57JGluc3RhbGxhdGlvbl9jb3VudH08L3NwYW4+IHBlb3BsZSB0cnlpbmcgeyR0aXRsZX0gcmlnaHQgbm93IVxudXNlckNvdW50Q29udGFpbmVyQWx0ID0gSnVzdCBsYXVuY2hlZCFcbmhpZ2hsaWdodFByaXZhY3kgPSBZb3VyIHByaXZhY3lcblxuIyBTaG93biB3aGVuIGFuIGV4cGVyaW1lbnQgcmVxdWlyZXMgYSB2ZXJzaW9uIG9mIEZpcmVmb3ggbmV3ZXIgdGhhbiB0aGUgdXNlcidzLlxuW1t1cGdyYWRlTm90aWNlXV1cbnVwZ3JhZGVOb3RpY2VUaXRsZSA9IHskdGl0bGV9IHJlcXVpcmVzIEZpcmVmb3ggeyRtaW5fcmVsZWFzZX0gb3IgbGF0ZXIuXG51cGdyYWRlTm90aWNlTGluayA9IEhvdyB0byB1cGRhdGUgRmlyZWZveC5cblxuIyBTaG93biB3aGlsZSB1bmluc3RhbGxpbmcgVGVzdCBQaWxvdC5cbltbdW5pbnN0YWxsXV1cbnJldGlyZURpYWxvZ1RpdGxlID0gVW5pbnN0YWxsIFRlc3QgUGlsb3Q/XG5yZXRpcmVNZXNzYWdlID0gQXMgeW91IHdpc2guIFRoaXMgd2lsbCBkaXNhYmxlIGFueSBhY3RpdmUgdGVzdHMsIHVuaW5zdGFsbCB0aGUgYWRkLW9uIGFuZCByZW1vdmUgeW91ciBhY2NvdW50IGluZm8gZnJvbSBvdXIgc2VydmVycy5cbnJldGlyZUVtYWlsTWVzc2FnZSA9IFRvIG9wdCBvdXQgb2YgZW1haWwgdXBkYXRlcywgc2ltcGx5IGNsaWNrIHRoZSA8ZW0+dW5zdWJzY3JpYmU8L2VtPiBsaW5rIG9uIGFueSBUZXN0IFBpbG90IGVtYWlsLlxucmV0aXJlU3VibWl0QnV0dG9uID0gUHJvY2VlZFxucmV0aXJlQ2FuY2VsQnV0dG9uID0gQ2FuY2VsXG5wYWdlVGl0bGVSZXRpcmVQYWdlID0gRmlyZWZveCBUZXN0IFBpbG90IC0gVW5pbnN0YWxsIFRlc3QgUGlsb3RcbnJldGlyZVBhZ2VQcm9ncmVzc01lc3NhZ2UgPSBTaHV0dGluZyBkb3duLi4uXG5yZXRpcmVQYWdlSGVhZGxpbmUgPSBUaGFua3MgZm9yIGZseWluZyFcbnJldGlyZVBhZ2VNZXNzYWdlID0gSG9wZSB5b3UgaGFkIGZ1biBleHBlcmltZW50aW5nIHdpdGggdXMuIDxicj4gQ29tZSBiYWNrIGFueSB0aW1lLlxucmV0aXJlUGFnZVN1cnZleUJ1dHRvbiA9IFRha2UgYSBxdWljayBzdXJ2ZXlcblxuIyBTaG93biB0byB1c2VycyBhZnRlciBpbnN0YWxsaW5nIFRlc3QgUGlsb3QgaWYgYSByZXN0YXJ0IGlzIHJlcXVpcmVkLlxuW1tyZXN0YXJ0SW50cm9dXVxucmVzdGFydEludHJvTGVhZCA9IFByZWZsaWdodCBjaGVja2xpc3RcbnJlc3RhcnRJbnRyb09uZSA9IFJlc3RhcnQgeW91ciBicm93c2VyXG5yZXN0YXJ0SW50cm9Ud28gPSBMb2NhdGUgdGhlIFRlc3QgUGlsb3QgYWRkLW9uXG5yZXN0YXJ0SW50cm9UaHJlZSA9IFNlbGVjdCB5b3VyIGV4cGVyaW1lbnRzXG5cbiMgU2hvd24gb24gYSBwYWdlIHByZXNlbnRlZCB0byB1c2VycyB0aHJlZSBkYXlzIGFmdGVyIGluc3RhbGxpbmcgdGhlaXIgZmlyc3QgZXhwZXJpbWVudC5cbltbc2hhcmVdXVxuc2hhcmVQcmltYXJ5ID0gTG92ZSBUZXN0IFBpbG90PyBIZWxwIHVzIGZpbmQgc29tZSBuZXcgcmVjcnVpdHMuXG5zaGFyZVNlY29uZGFyeSA9IG9yIGp1c3QgY29weSBhbmQgcGFzdGUgdGhpcyBsaW5rLi4uXG5zaGFyZUVtYWlsID0gRS1tYWlsXG5zaGFyZUNvcHkgPSBDb3B5XG5cbiMgU2hvd24gb24gcGFnZXMgb2YgcmV0aXJlZCBleHBlcmltZW50cy5cbltbZW9sXV1cbmVvbE1lc3NhZ2UgPSA8c3Ryb25nPlRoaXMgZXhwZXJpbWVudCBpcyBlbmRpbmcgb24geyRjb21wbGV0ZWREYXRlfTwvc3Ryb25nPi48YnIvPjxici8+QWZ0ZXIgdGhlbiB5b3Ugd2lsbCBzdGlsbCBiZSBhYmxlIHRvIHVzZSB7JHRpdGxlfSBidXQgd2Ugd2lsbCBubyBsb25nZXIgYmUgcHJvdmlkaW5nIHVwZGF0ZXMgb3Igc3VwcG9ydC5cbmVvbERpc2FibGVNZXNzYWdlID0gVGhlIHskdGl0bGV9IGV4cGVyaW1lbnQgaGFzIGVuZGVkLiBPbmNlIHlvdSB1bmluc3RhbGwgaXQgeW91IHdvbid0IGJlIGFibGUgdG8gcmUtaW5zdGFsbCBpdCB0aHJvdWdoIFRlc3QgUGlsb3QgYWdhaW4uXG5jb21wbGV0ZWREYXRlTGFiZWwgPSBFeHBlcmltZW50IEVuZCBEYXRlOiA8Yj57JGNvbXBsZXRlZERhdGV9PC9iPlxuXG4jIEEgd2FybmluZyBzaG93biB0byB1c2VycyBsb29raW5nIGF0IGV4cGVyaW1lbnRzIGluY29tcGF0aWJsZSB3aXRoIGFkZC1vbnMgdGhleSBhbHJlYWR5IGhhdmUgaW5zdGFsbGVkLlxuW1tpbmNvbXBhdGlibGVdXVxuaW5jb21wYXRpYmxlSGVhZGVyID0gVGhpcyBleHBlcmltZW50IG1heSBub3QgYmUgY29tcGF0aWJsZSB3aXRoIGFkZC1vbnMgeW91IGhhdmUgaW5zdGFsbGVkLlxuaW5jb21wYXRpYmxlU3ViaGVhZGVyID0gV2UgcmVjb21tZW5kIDxhPmRpc2FibGluZyB0aGVzZSBhZGQtb25zPC9hPiBiZWZvcmUgYWN0aXZhdGluZyB0aGlzIGV4cGVyaW1lbnQ6XG5cbiMgQSBmb3JtIHByb21wdGluZyB0aGUgdXNlciB0byBzaWduIHVwIGZvciB0aGUgVGVzdCBQaWxvdCBOZXdzbGV0dGVyLlxuW1tuZXdzbGV0dGVyRm9ybV1dXG5uZXdzbGV0dGVyRm9ybUVtYWlsUGxhY2Vob2xkZXIgPVxuICAgIFtodG1sL3BsYWNlaG9sZGVyXSBZb3VyIGVtYWlsIGhlcmVcbm5ld3NsZXR0ZXJGb3JtRGlzY2xhaW1lciA9IFdlIHdpbGwgb25seSBzZW5kIHlvdSBUZXN0IFBpbG90LXJlbGF0ZWQgaW5mb3JtYXRpb24uXG5uZXdzbGV0dGVyRm9ybVByaXZhY3lOb3RpY2UgPSBJJ20gb2theSB3aXRoIE1vemlsbGEgaGFuZGxpbmcgbXkgaW5mbyBhcyBleHBsYWluZWQgaW4gPGE+dGhpcyBwcml2YWN5IG5vdGljZTwvYT4uXG5uZXdzbGV0dGVyRm9ybVN1Ym1pdEJ1dHRvbiA9IFNpZ24gVXAgTm93XG5uZXdzbGV0dGVyRm9ybVN1Ym1pdEJ1dHRvblN1Ym1pdHRpbmcgPSBTdWJtaXR0aW5nLi4uXG5cbiMgQSBzZWN0aW9uIG9mIHRoZSBmb290ZXIgY29udGFpbmluZyBhIG5ld3NsZXR0ZXIgc2lnbnVwIGZvcm0uXG5bW25ld3NsZXR0ZXJGb290ZXJdXVxubmV3c2xldHRlckZvb3RlckVycm9yID0gVGhlcmUgd2FzIGFuIGVycm9yIHN1Ym1pdHRpbmcgeW91ciBlbWFpbCBhZGRyZXNzLiBUcnkgYWdhaW4/XG5uZXdzbGV0dGVyRm9vdGVySGVhZGVyID0gU3RheSBJbmZvcm1lZFxubmV3c2xldHRlckZvb3RlckJvZHkgPSBGaW5kIG91dCBhYm91dCBuZXcgZXhwZXJpbWVudHMgYW5kIHNlZSB0ZXN0IHJlc3VsdHMgZm9yIGV4cGVyaW1lbnRzIHlvdSd2ZSB0cmllZC5cbm5ld3NsZXR0ZXJGb290ZXJTdWNjZXNzSGVhZGVyID0gVGhhbmtzIVxubmV3c2xldHRlckZvb3RlclN1Y2Nlc3NCb2R5ID0gSWYgeW91IGhhdmVuJ3QgcHJldmlvdXNseSBjb25maXJtZWQgYSBzdWJzY3JpcHRpb24gdG8gYSBNb3ppbGxhLXJlbGF0ZWQgbmV3c2xldHRlciB5b3UgbWF5IGhhdmUgdG8gZG8gc28uIFBsZWFzZSBjaGVjayB5b3VyIGluYm94IG9yIHlvdXIgc3BhbSBmaWx0ZXIgZm9yIGFuIGVtYWlsIGZyb20gdXMuXG5cbiMgQSB3YXJuaW5nIHNob3duIHRvIHVzZXJzIHZpZXdpbmcgYW4gZXhwZXJpbWVudCB0aGF0IGlzIG9ubHkgYXZhaWxhYmxlIGluIEVuZ2xpc2guXG5bW2xvY2FsZVdhcm5pbmddXVxubG9jYWxlV2FybmluZ1RpdGxlID0gVGhpcyBleHBlcmltZW50IGlzIG9ubHkgYXZhaWxhYmxlIGluIEVuZ2xpc2guXG5sb2NhbGVXYXJuaW5nU3VidGl0bGUgPSBZb3UgY2FuIHN0aWxsIGVuYWJsZSBpdCBpZiB5b3UgbGlrZS5cblxuIyBBbiBhbHRlcm5hdGUgc3BsYXNoIHBhZ2Ugc2hvd24gdG8gdXNlcnMgd2hvIGhhdmUgaGFkIFRlc3QgUGlsb3QgaW5zdGFsbGVkIGZvciBzb21lIHRpbWUsIGJ1dCBoYXZlIG5vIGV4cGVyaW1lbnRzIGluc3RhbGxlZC5cbltbZXhwZXJpbWVudHNMaXN0Tm9uZUluc3RhbGxlZF1dXG5leHBlcmltZW50c0xpc3ROb25lSW5zdGFsbGVkSGVhZGVyID0gTGV0J3MgZ2V0IHRoaXMgYmFieSBvZmYgdGhlIGdyb3VuZCFcbmV4cGVyaW1lbnRzTGlzdE5vbmVJbnN0YWxsZWRTdWJoZWFkZXIgPSBSZWFkeSB0byB0cnkgYSBuZXcgVGVzdCBQaWxvdCBleHBlcmltZW50PyBTZWxlY3Qgb25lIHRvIGVuYWJsZSwgdGFrZSBpdCBmb3IgYSBzcGluLCBhbmQgbGV0IHVzIGtub3cgd2hhdCB5b3UgdGhpbmsuXG5leHBlcmltZW50c0xpc3ROb25lSW5zdGFsbGVkQ1RBID0gTm90IGludGVyZXN0ZWQ/IDxhPkxldCB1cyBrbm93IHdoeTwvYT4uXG5cbiMgU2hvd24gdG8gdXNlcnMgd2hvIGRvIG5vdCBoYXZlIEphdmFTY3JpcHQgZW5hYmxlZC5cbltbbm9zY3JpcHRdXVxubm9TY3JpcHRIZWFkaW5nID0gVWggb2guLi5cbm5vU2NyaXB0TWVzc2FnZSA9IFRlc3QgUGlsb3QgcmVxdWlyZXMgSmF2YVNjcmlwdC48YnI+U29ycnkgYWJvdXQgdGhhdC5cbm5vU2NyaXB0TGluayA9IEZpbmQgb3V0IHdoeVxuXG4jIFRleHQgb2YgYSBidXR0b24gdG8gdG9nZ2xlIHZpc2liaWxpdHkgb2YgYSBsaXN0IG9mIHBhc3QgZXhwZXJpbWVudHMuXG5bW3Bhc3RFeHBlcmltZW50c11dXG52aWV3UGFzdEV4cGVyaW1lbnRzID0gVmlldyBQYXN0IEV4cGVyaW1lbnRzXG5oaWRlUGFzdEV4cGVyaW1lbnRzID0gSGlkZSBQYXN0IEV4cGVyaW1lbnRzXG4iLCJjb250ZXh0IjoiIiwiY29uZmlnIjp7ImVzY2FwZUh0bWwiOnRydWUsImRpciI6Imx0ciIsImxhbmciOiJlbi1VUyIsImZpeHR1cmUiOiJkZWZhdWx0In19).

Very handy tool, thanks for sharing!